elastic/kibana
# [Visualize] Renders no data component if there is no ES data or dataview

## Summary

Part of https://github.com/elastic/kibana/issues/132146

It implements the no data component to the visualize library.
<img width="1335" alt="image" src="https://user-images.githubusercontent.com/17003240/168795584-3b218237-7c91-4355-861b-9edfa7e1e307.png">

This component is loaded when:
- there is no es data
- there is ES data but no dataviews

### How to test
- Start a new kibana instance (with the functional test server for example) with no data. Go to visualize. You will see the integrations component
- Add some sample data, go to dataviews management page and remove the dataviews. You will see the no dataviews component.

### Checklist
